src/messages/MMgrConfigure.h

```diff
@@ -33,7 +34,9 @@ class MMgrConfigure : public Message {
   // Default 0 means if unspecified will include all stats
   uint32_t stats_threshold = 0;
 
-  std::map<OSDPerfMetricQuery, OSDPerfMetricLimits> osd_perf_metric_queries;
+  boost::optional<std::map<OSDPerfMetricQuery, OSDPerfMetricLimits>> osd_perf_metric_queries;
```

Comment:
true -- I had made changes to `MgrClient::_send_configure()` to use `boost::optional` which I reverted since it wasn't backward compatible. Forgot to revert this bit.